### PR TITLE
DISTMYSQL-94 Added README, License and Contributing guides

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   CONTRIBUTING.md
	new file:   LICENSE
	modified:   README.md
	new file:   source/_static/images/PDMYSQL_links.png

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Contributing Guide
+
+Thank you for deciding to contribute and help us improve Percona Distribution for MySQL documentation!
+
+We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
```
Comment (LenzGr) [inline CONTRIBUTING.md]: I suggest to use the rendered online version instead: https://percona.community/contribute/coc/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+git checkout -b <my_changes>
+```
+
+5. Make changes
+6. Commit your changes
+7. Open a pull request to Percona
+
+### Building the documentation
+
+To verify how your changes look, generate the static site with the documentation. This process is called *building*. You can do it in these ways:
```
Comment (LenzGr) [inline CONTRIBUTING.md:79]:
```suggestion
To verify the layout of your changes, generate the static site with the documentation. This process is called *building*. You can do it in these ways:
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (nastena1606) [reply CONTRIBUTING.md]: Fixed

Comment (LenzGr) [reply CONTRIBUTING.md]: Somehow that change does not seem to have made it into your last push?

Comment (nastena1606) [reply CONTRIBUTING.md]: Strange. Now it must look OK

**Review by LenzGr — APPROVED** [review]
LGTM! Thank you.

**Pull request merged**